frontend/src/views/Question.vue

```diff
@@ -58,7 +58,7 @@ export default {
         // this.answerFormResponse.style = {'form-success': false, 'form-failure': true}
         return
       }
-      this.$store.dispatch('questions/postAnswer', {form: answerForm, id: this.question.id})
+      this.$store.dispatch('questions/postAnswer', {form: answerForm, code: this.code ,id: this.question.id})
```

Comment:
I'm not of a fan of:
```
code: this.code ,id:
```
Should be:
```
code: this.code, id:
```
IMPORTANT CHANGE

Reply:
Fair.